src/Diagnostics.ModelsAndUtils/Utilities/AzureSupportCenterInsightUtilities.cs

```diff
@@ -52,14 +52,13 @@ internal static AzureSupportCenterInsight CreateInsight<TResource>(string title,
                 InsightFriendlyName = category,
                 IssueCategory = category,
                 IssueSubCategory = category,
-                Description = description,
+                Description = description?? new Text("Please submit a feedback for the detector author via AppLens to supply a description for this insight.", false),
                 RecommendedAction = new RecommendedAction()
                 {
                     Id = Guid.NewGuid(),
-                    Text = recommendedAction
+                    Text = recommendedAction?? new Text("Please submit a feedback for the detector author via AppLens to populate recommended action.", false)
```

Comment:
Create a const string maybe defined at the top to hold this string value.

Reply:
Good point, updated the PR.